dbt/models/default/default.pinval_assessment_card.sql

```diff
@@ -0,0 +1,285 @@
+    ac.prox_nearest_road_arterial_speed_limit,
+    ac.prox_nearest_road_arterial_surface_type,
+    ac.prox_nearest_road_collector_daily_traffic,
+    ac.prox_nearest_road_collector_dist_ft,
+    ac.prox_nearest_road_collector_lanes,
+    ac.prox_nearest_road_collector_speed_limit,
+    ac.prox_nearest_road_collector_surface_type,
+    ac.prox_nearest_road_highway_daily_traffic,
+    ac.prox_nearest_road_highway_dist_ft,
+    ac.prox_nearest_road_highway_lanes,
+    ac.prox_nearest_road_highway_speed_limit,
+    ac.prox_nearest_road_highway_surface_type,
+    ac.prox_nearest_stadium_dist_ft,
+    ac.time_sale_roll_mean_town_t0_w1,
+    ac.time_sale_roll_mean_town_t0_w2,
+    ac.time_sale_roll_mean_town_t0_w3,
+    ac.time_sale_roll_mean_town_t1_w1,
+    ac.time_sale_roll_mean_town_t1_w2,
+    ac.time_sale_roll_mean_town_t1_w3,
+    ac.time_sale_roll_mean_town_t2_w1,
+    ac.time_sale_roll_mean_town_t2_w2,
+    ac.time_sale_roll_mean_town_t2_w3,
+    ac.time_sale_roll_mean_nbhd_t0_w1,
+    ac.time_sale_roll_mean_nbhd_t0_w2,
+    ac.time_sale_roll_mean_nbhd_t0_w3,
+    ac.time_sale_roll_mean_nbhd_t1_w1,
+    ac.time_sale_roll_mean_nbhd_t1_w2,
+    ac.time_sale_roll_mean_nbhd_t1_w3,
+    ac.time_sale_roll_mean_nbhd_t2_w1,
+    ac.time_sale_roll_mean_nbhd_t2_w2,
+    ac.time_sale_roll_mean_nbhd_t2_w3,
+    ac.permit_amount,
+    ac.permit_binary,
+    ac.meta_pin_num_cards,
+    ac."floor", -- this is a reserved word in SQL
+    ac.time_sale_roll_mean_nbhd_sf_t0_w1,
+    ac.time_sale_roll_mean_nbhd_sf_t0_w2,
+    ac.time_sale_roll_mean_nbhd_sf_t0_w3,
+    ac.time_sale_roll_mean_nbhd_sf_t1_w1,
+    ac.time_sale_roll_mean_nbhd_sf_t1_w2,
+    ac.time_sale_roll_mean_nbhd_sf_t1_w3,
+    ac.time_sale_roll_mean_nbhd_sf_t2_w1,
+    ac.time_sale_roll_mean_nbhd_sf_t2_w2,
+    ac.time_sale_roll_mean_nbhd_sf_t2_w3,
+    ac.time_sale_roll_mean_nbhd_condo_t0_w1,
+    ac.time_sale_roll_mean_nbhd_condo_t0_w2,
+    ac.time_sale_roll_mean_nbhd_condo_t0_w3,
+    ac.time_sale_roll_mean_nbhd_condo_t1_w1,
+    ac.time_sale_roll_mean_nbhd_condo_t1_w2,
+    ac.time_sale_roll_mean_nbhd_condo_t1_w3,
+    ac.time_sale_roll_mean_nbhd_condo_t2_w1,
+    ac.time_sale_roll_mean_nbhd_condo_t2_w2,
+    ac.time_sale_roll_mean_nbhd_condo_t2_w3,
+    ac.meta_bldg_roll_mean,
+    ac.meta_pin10_bldg_roll_mean,
+    ac.meta_pin10_bldg_roll_pct_sold,
+    ac.year,
+    ac.run_id,
+    ac.township_code,
+
+    -- This is just a fancy way to capitalize the first letter of each word
+    -- since `initcap` doesn't seem to work in athena
+    ap.pred_pin_final_fmv_round,
+    ARRAY_JOIN(
+        TRANSFORM(
+            SPLIT(LOWER(ap.loc_property_address), ' '),
+            x -> CONCAT(UPPER(SUBSTR(x, 1, 1)), SUBSTR(x, 2))
+        ),
+        ' '
+    ) AS property_address,
+
+    school.school_elementary_district_name
+        AS loc_school_elementary_district_name,
+    school.school_secondary_district_name
+        AS loc_school_secondary_district_name
```

Comment:
Assessment pin columns added here to factor out the old `assessment_pin_df` pull from the quarto doc, which speeds up the program generation